internal/plan/plan.go

```diff
@@ -105,6 +149,15 @@ const (
 	ReplaceOverride ServiceOverride = "replace"
 )
 
+type ServiceAction string
+
+const (
+	ActionUnset      ServiceAction = ""
+	ActionRestart    ServiceAction = "restart"
+	ActionExitPebble ServiceAction = "exit-pebble"
+	ActionLog        ServiceAction = "log"
```

Comment:
I find this one a bit awkward as well. I mean, good logging should be in place regardless and is not analogous to the other options. But this is easy for us to tune before we're done.

Reply:
Per discussion, updated to just "ignore".